test/unit/rubygem_test.rb

```diff
@@ -338,24 +338,23 @@ class RubygemTest < ActiveSupport::TestCase
 
     context "with a user" do
       setup do
-        @rubygem.save
-        @user = create(:user)
+        @user = build(:user)
```

Comment:
I think we have this discussion before @nateberkopec, but again - this change probably make it faster (even not really noticeable), but far away from real case scenario. In production those methods are called to query data stored in DB.